src/styles/index.css

```diff
@@ -0,0 +1,393 @@
+  content: "18";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(19)::before {
+  content: "19";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(20)::before {
+  content: "20";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(21)::before {
+  content: "21";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(22)::before {
+  content: "22";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(23)::before {
+  content: "23";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(24)::before {
+  content: "24";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(25)::before {
+  content: "25";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(26)::before {
+  content: "26";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(27)::before {
+  content: "27";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(28)::before {
+  content: "28";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(29)::before {
+  content: "29";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(30)::before {
+  content: "30";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(31)::before {
+  content: "31";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
```

Comment:
There is a lot of repetitive code for the `.calendar__day::before` pseudo-elements. To follow the DRY (Don't Repeat Yourself) principle, you should use a common class for the shared styles (like `font-family`, `font-size`, `position`, `top`, `left`, `transform`) and only change the `content` property for each day. This will make the code more maintainable and easier to read.